### ISSUE TITLE
Failing attempt to replace image 

### ISSUE THREAD

**Issue by imanhodjaev** [wagtail/wagtail#4980]
### Issue Summary
Problem occurs when replacing images, sometimes Wagtail admin seems to upload an image and very often fail w/ the following exception, is there anything to configure or do properly when replacing images?

```
ValueError: I/O operation on closed file.
ERROR 2019-01-04 10:15:22,366 log 36751 123145484890112 Internal Server Error: /admin/images/1/
Traceback (most recent call last):
```

### Steps to Reproduce

1. `wagtail start myproject`
2. Upload an image
3. Try to replace image with new one

I've been also trying to debug it and at the moment looking at https://github.com/wagtail/wagtail/blob/master/wagtail/images/views/images.py#L101
Just before `form.is_valid()` `form.image.file` is not seekable. After setting breakpoints and debugging I tried to call `form.is_valid()` twice and on second call it was `True`.

### Technical details

* Python version: 3.6.7,
* Wagtail version: 2.4.0,
* Django version: 2.1.4,
* Chrome: 71,
* OS: MacOS.

Full exception

```
[04/Jan/2019 14:33:22] "POST /admin/images/1/ HTTP/1.1" 500 157806
[04/Jan/2019 14:33:23] "GET /admin/images/1/ HTTP/1.1" 200 14585
Internal Server Error: /admin/images/1/
Traceback (most recent call last):
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/views/decorators/cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/wagtail/admin/urls/__init__.py", line 102, in wrapper
    return view_func(request, *args, **kwargs)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/wagtail/admin/decorators.py", line 34, in decorated_view
    return view_func(request, *args, **kwargs)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/wagtail/admin/utils.py", line 148, in wrapped_view_func
    return view_func(request, *args, **kwargs)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/wagtail/images/views/images.py", line 101, in edit
    if form.is_valid():
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/forms/forms.py", line 185, in is_valid
    return self.is_bound and not self.errors
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/forms/forms.py", line 180, in errors
    self.full_clean()
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/forms/forms.py", line 383, in full_clean
    self._post_clean()
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/forms/models.py", line 398, in _post_clean
    self.instance = construct_instance(self, self.instance, opts.fields, opts.exclude)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/forms/models.py", line 63, in construct_instance
    f.save_form_data(instance, cleaned_data[f.name])
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/db/models/fields/files.py", line 317, in save_form_data
    setattr(instance, self.name, data or '')
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/db/models/fields/files.py", line 346, in __set__
    self.field.update_dimension_fields(instance, force=True)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/db/models/fields/files.py", line 449, in update_dimension_fields
    width = file.width
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/core/files/images.py", line 19, in width
    return self._get_image_dimensions()[0]
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/core/files/images.py", line 29, in _get_image_dimensions
    self._dimensions_cache = get_image_dimensions(self, close=close)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/django/core/files/images.py", line 59, in get_image_dimensions
    p.feed(data)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/PIL/ImageFile.py", line 399, in feed
    im = Image.open(fp)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/PIL/Image.py", line 2672, in open
    im = _open_core(fp, filename, prefix)
  File "Envs/publizist-V24PH7XT/lib/python3.6/site-packages/PIL/Image.py", line 2657, in _open_core
    fp.seek(0)
ValueError: I/O operation on closed file.
```

**Comment by imanhodjaev** [comment]
Found out that the issue only comes in Wagtail 2.4.0

**Comment by imanhodjaev** [comment]
Problem resolved with Django 2.1.5 however I am not sure what has caused the issue, anyway now we can use the latest Wagtail and Django without facing this issue again.
